whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/SchedulerConfig.java

```diff
@@ -17,9 +18,9 @@
 import javax.sql.DataSource;
 
 @Configuration
-// @EnableAspectJAutoProxy(proxyTargetClass = true)
+@EnableAspectJAutoProxy(proxyTargetClass = true)
 @EnableScheduling
-@EnableSchedulerLock(defaultLockAtMostFor="23h", mode = AdviceMode.ASPECTJ)
```

Comment:
Also is it better to be explicit?

Reply:
I wanted it to be as it was before spring upgrade.